x-pack/plugins/security_solution/public/timelines/store/timeline/reducer.ts

```diff
@@ -135,24 +136,34 @@ export const timelineReducer = reducerWithInitialState(initialTimelineState)
         timelineType = TimelineType.default,
         filters,
       }
-    ) => ({
-      ...state,
-      timelineById: addNewTimeline({
-        columns,
-        dataProviders,
-        dateRange,
-        filters,
-        id,
-        itemsPerPage,
-        kqlQuery,
-        sort,
-        show,
-        showCheckboxes,
-        showRowRenderers,
-        timelineById: state.timelineById,
-        timelineType,
-      }),
-    })
+    ) => {
+      const templateTimelineInfo =
```

Comment:
I am wondering if this code belong inside of the helper file with the function `addNewTimeline`